core/templates/dev/head/pages/preferences-page/preferences-page.directive.html

```diff
@@ -252,4 +252,10 @@
     position: relative;
     text-align: center;
   }
+  .protractor-test-user-bio {
```

Comment:
Hi @doc-han, have you tried adding "form-control" class for the textarea?
see: https://getbootstrap.com/docs/4.0/components/forms/